polkadot-protocol/architecture/polkadot-chain/overview.md

```diff
@@ -1,4 +1,371 @@
+
+4. **Resilient infrastructure**: Decentralized and scalable, Polkadot ensures ongoing support for development and community initiatives via its on-chain [treasury](https://wiki.polkadot.network/docs/learn-polkadot-opengov-treasury){target=\_blank} and governance.
+
+5. **Rapid L1 development**: The [Polkadot SDK](link to Polkadot SDK page in polkadot protocol section) allows fast, flexible creation and deployment of Layer-1 chains.
+
+6. **Cultivating the next generation of Web3 developers**: Polkadot supports the growth of Web3 core developers through initiatives such as:
+
+    - [Polkadot Blockchain Academy](https://polkadot.com/blockchain-academy){target=\_blank}
+    - [Polkadot Alpha Program](https://polkadot.com/alpha-program){target=\_blank}
+    - [Polkadot Developer Heroes Program](https://wiki.polkadot.network/docs/dev-heroes){target=\_blank}
+    - [EdX Courses](https://www.edx.org/school/web3x){target=\_blank}
+    - Rust and Substrate Courses (coming soon)
+
+### High Level Architecture
+
+Polkadot features a chain that serves as the central component of the system. This chain is depicted as a ring, encircled by several Polkadot chains that are connected to it. 
+
+According to Polkadot's design, any chain that can compile to WebAssembly (Wasm) and complies with the  chain's API can integrate into the Polkadot network as a parachain.
+
+Here’s a high-level overview of the Polkadot protocol architecture
+
+![](/images/polkadot-protocol/architecture/polkadot-chain/overview/relay-chain-architecture.webp)
+
+Polkadot chains propose blocks to the Polkadot validators, who perform checks for availability and validity before finalizing them. With the  chain providing security, collators — full nodes of Polkadot chains — can focus on their tasks without needing strong incentives.
+
+The [Cross-Consensus Messaging Format (XCM)](add link to XCM page){target=\_blank} allows Polkadot chains to exchange messages freely, leveraging the  chain's security for trust-free communication.
+
+In order to interact with chains that want to use their own finalization process (e.g. Bitcoin), Polkadot has [bridges](add link to bridges pages, Snowbridge, etc){target=\_blank}  that offer two-way compatibility, meaning that transactions can be made between different Polkadot chains.
+
+### Polkadot's Additional Functionalities
+
+The Polkadot  chain oversees crowdloans and auctions. Chain cores were leased through auctions for three-month periods, up to a maximum of two years. 
+
+Crowdloans enabled users to securely lend funds to teams for lease deposits in exchange for pre-sale tokens, which is the only way to access slots on Polkadot 1.0.
+
+!!!note
+    Auction are deprecated in favor of Coretime
+
+Additionally, the  chain handles [staking](add link to PoS page){target=\_blank}, [accounts](add a link to accounts pages){target=\_blank}, balances, and [governance](add a link to govenance page){target=\_blank}.
```

Comment:
I would write a brief explanation of coretime first because it is the current mechanism being used, and then you can add crowdloans and auctions as a note

Reply:
Explanation added